lib/solidus_stripe/configuration.rb

```diff
@@ -1,10 +1,24 @@
 # frozen_string_literal: true
 
+require "stripe/webhook"
+
 module SolidusStripe
   class Configuration
-    # Define here the settings for this extension, e.g.:
-    #
-    # attr_accessor :my_setting
+    # @!attribute [rw] webhook_events
+    #  @return [Array<Symbol>] stripe events to handle. You also need to
+    #  register them in the Stripe dashboard. For an event `:foo`, a matching
+    #  `:"stripe.foo"` event will be registered in {Spree::Bus}.
+    attr_accessor :webhook_events
```

Comment:
Can user remove events that we need for the extension with this preference?

Reply:
No, they can't, as we add them into the `SolidusStripe::Webhook::Event::CORE_EVENTS` constant.